components/brave_ads/core/public/ads_client/ads_client.h

```diff
@@ -32,7 +32,7 @@ class ADS_EXPORT AdsClient {
   // Called to remove an ads client observer.
   virtual void RemoveObserver(AdsClientNotifierObserver* observer) = 0;
 
-  // Called to bind pending ads client observers.
+  // Called to fire all pending observer events.
```

Comment:
nit: `Called to notify pending observers.`

Reply:
Fixed